tests/conftest.py

```diff
@@ -9,8 +9,11 @@
 async def proxy():
     """Fixture returning a configured Traefik Proxy"""
     proxy = TraefikEtcdProxy()
-    await proxy.start()
-    yield proxy
+    try:
+        await proxy.start()
+        yield proxy
+    finally:
+        await proxy.stop()
```

Comment:
This should mean that the `proxy.stop` can be removed from the tests that were calling it.